service/controller/app/v1/resource/status/create.go

```diff
@@ -0,0 +1,53 @@
+package status
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	cr, err := key.ToCustomResource(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	name := key.AppName(cr)
+
+	g8sClient, err := r.kubeConfig.NewG8sClientForApp(ctx, cr)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("getting status for chart %#q", name))
```

Comment:
```suggestion
	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("getting status for app %#q", name))
```

This logging isn't clear to me. We need to get the status the `chart` CR status and use it to set the status for the `app` CR.

Maybe we should also adding logging for `finding status for chart %#q"` and `found status for chart %#q"`
https://github.com/giantswarm/fmt/blob/master/go/log_messages.md#fact-gathering WDYT?

Reply:
I'll change this log `finding status for chart %#q` and add one more log as `found status for chart %#`, in that way we are more clear I guess.